e2e/process-services-cloud/process-filter-results.e2e.ts

```diff
@@ -0,0 +1,304 @@
+
+        processInstancesService = new ProcessInstancesService(apiService);
+        await processInstancesService.createProcessInstance(processDefinition.entry.key, candidateBaseApp);
+
+        runningProcessInstance = await processInstancesService.createProcessInstance(processDefinition.entry.key, candidateBaseApp, {
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+
+        anotherProcessInstance = await processInstancesService.createProcessInstance(anotherProcessDefinition.entry.key, candidateBaseApp, {
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+
+        suspendProcessInstance = await processInstancesService.createProcessInstance(processDefinition.entry.key, candidateBaseApp, {
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+        await processInstancesService.suspendProcessInstance(suspendProcessInstance.entry.id, candidateBaseApp);
+
+        completedProcess = await processInstancesService.createProcessInstance(processDefinition.entry.key, candidateBaseApp, {
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+        queryService = new QueryService(apiService);
+
+        const task = await queryService.getProcessInstanceTasks(completedProcess.entry.id, candidateBaseApp);
+        tasksService = new TasksService(apiService);
+        const claimedTask = await tasksService.claimTask(task.list.entries[0].entry.id, candidateBaseApp);
+        await tasksService.completeTask(claimedTask.entry.id, candidateBaseApp);
+
+        await settingsPage.setProviderBpmSso(
+            browser.params.config.bpmHost,
+            browser.params.config.oauth2.host,
+            browser.params.config.identityHost);
+        loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+        await LocalStorageUtil.setConfigField('adf-edit-process-filter', JSON.stringify(editProcessFilterConfigFile));
+        await LocalStorageUtil.setConfigField('adf-cloud-process-list', JSON.stringify(processListCloudConfigFile));
+        done();
+    });
+
+    afterAll(async (done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+        await identityService.deleteIdentityUser(testUser.idIdentityService);
+        done();
+    });
+
+    beforeEach(() => {
+        navigationBarPage.navigateToProcessServicesCloudPage();
+        appListCloudComponent.checkApsContainer();
+        appListCloudComponent.goToApp(candidateBaseApp);
+        tasksCloudDemoPage.taskListCloudComponent().checkTaskListIsLoaded();
+        processCloudDemoPage.clickOnProcessFilters();
+    });
+
+    it('[C306887] Should be able to filter by appName', async () => {
```

Comment:
From my point of view, one more check should be added related to the fact that a process instance started on a different app is not displayed. (Note: Step 4 of this test case from test rail duplicates C306889, please delete it from test rail.)